src/main/java/com/playmonumenta/networkchat/NetworkChatPlugin.java

```diff
@@ -110,6 +110,18 @@ public void onLoad() {
 			MessagingUtils.sendStackTrace(Bukkit.getConsoleSender(), e);
 		}
 
+		try {
+			mGlobalChatFilter.addFilter(Bukkit.getConsoleSender(),
+				"Spoiler",
+				false,
+				"\\|\\|(.+)\\|\\|",
```

Comment:
Consider using `.+?` for non-greedy matching to properly support multiple spoilers in a single message

Reply:
Good call, uploading that change.